tests/sles4sap/patterns.pm

```diff
@@ -51,6 +55,9 @@ sub run {
         die "SAP zypper pattern [$pattern] info check failed"
           unless ($output =~ /i\+\s\|\spatterns-$pattern\s+\|\spackage\s\|\sRequired/);
     }
+
+    # Enable packagekit
+    systemctl '--now unmask packagekit.service';
```

Comment:
any good reason to enable it ?

Reply:
Just to return the system to its previous state. But I don't think there are further tests that will require `packagekit` enabled, so I better drop it.